package.json

```diff
@@ -45,7 +46,7 @@
     "font-awesome": "^4.7.0",
     "lodash.uniqby": "^4.7.0",
     "mailto-link": "^1.0.0",
-    "prop-types": "^15.8.1",
+    "prop-types": "^15.7.2",
```

Comment:
Why do we downgrade here?

Reply:
Npm `8.7` changes automatically for some reason. I reverted back